README.md

```diff
@@ -107,6 +107,7 @@ The below devices are fully feature supported by OpenRazer which means all avail
 | --------------------------------------------- | ----------- |
 | Razer Firefly Hyperflux                       |  1532:0068  |
 | Razer Firefly                                 |  1532:0C00  |
+| Razer Firefly v2                              |  1532:0C04  |
 | Razer Goliathus                               |  1532:0C01  |
 | Razer Goliathus Extended                      |  1532:0C02  |
```

Comment:
Please put the device after the Goliathus extended (so it's sorted by usb pid)

Reply:
Put it after goliathus extended now